### ISSUE TITLE
Bug: Unexpected type of JSON object member with key `subject_types_supported`

### ISSUE THREAD

**Issue by Olatunji-Longe** [authorizerdev/authorizer#407]
**Version:** [1.1.70](https://github.com/authorizerdev/authorizer/releases/tag/1.1.70)

Related Issue: https://github.com/authorizerdev/authorizer/issues/304

Encountered this problem when I tried to use **_Authorizer_** with my java springboot client service ... though I assume the fix should be quite simple - Ref: [My comment on this PR](https://github.com/authorizerdev/authorizer/pull/332/files#r1367158957);

when I navigate to my authorizer instance's `https://my-authorizer-instance/.well-known/openid-configuration, I see `subject_types_supported` node as just a **_string_**;
```
{
    . . . 
    
    "subject_types_supported": "public",
    
    . . .
}
``` 

However the expectation is for it to be an **_array_** . See [OpenID Specs](https://openid.net/specs/openid-connect-core-1_0.html#:~:text=The%20OpenID%20Provider%27s%20Discovery%20document%20SHOULD%20list%20its%20supported%20Subject%20Identifier%20types%20in%20the%20subject_types_supported%20element.%20If%20there%20is%20more%20than%20one%20type%20listed%20in%20the%20array%2C%20the%20Client%20MAY%20elect%20to%20provide%20its%20preferred%20identifier%20type%20using%20the%20subject_type%20parameter%20during%20Registration.) which requires `subject_types_supported`  to be an array. See [example here](https://openid.net/specs/openid-connect-core-1_0.html#:~:text=phone%22%5D%2C%0A%20%20%20%22response_types_supported%22%3A%0A%20%20%20%20%20%5B%22id_token%22%5D%2C%0A%20%20%20%22-,subject_types_supported,-%22%3A%0A%20%20%20%20%20%5B%22pairwise%22%5D%2C%0A%20%20%20%22id_token_signing_alg_values_supported%22%3A%0A%20%20%20%20%20%5B%22RS256)
```
{
    . . . 
    
    "subject_types_supported": ["public"],
    
    . . .
}
``` 

Hence the error message;
```
Unexpected type of JSON object member with key subject_types_supported
```

**Steps To Reproduce**

Create a spring boot resource client application (you can easily [clone this sample](https://github.com/spring-projects/spring-authorization-server/tree/main/samples/demo-client)) and point the issuer-uri config to Authorizer's issuer uri

```
spring:
  security:
    oauth2:
      client:
        provider:
          spring:
            issuer-uri: https://my-authorizer-instance
```
Start the springboot application and see the error in the logs.

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

- OS: Ubuntu
- Browser: Chrome
- Version: 118.0.5993.70

**Additional context**

**Comment by Olatunji-Longe** [comment]
I [opened a PR](https://github.com/authorizerdev/authorizer/pull/408) to fix this. Tested on a local instance of Authorizer to be working well with a java springboot client

**Comment by lakhansamani** [comment]
Thanks @Olatunji-Longe will be part of 1.1.71. I will release it by tomorrow

**Comment by lakhansamani** [comment]
Released as part of 1.1.71
